types/src/transaction/transaction_v1.rs

```diff
@@ -650,6 +650,12 @@ impl GasLimited for TransactionV1 {
                             TransactionEntryPoint::ChangeBidPublicKey => {
                                 costs.auction_costs().change_bid_public_key
                             }
+                            TransactionEntryPoint::AddReservation => {
+                                todo!()
+                            }
+                            TransactionEntryPoint::CancelReservation => {
+                                todo!()
+                            }
```

Comment:
you need to add cost fields to the chainspec auction costs table

you can set the initial value for both to be the same as `redelegate` and we can adjust as necessary.